packages/blocks/src/page-block/utils/bind-hotkey.ts

```diff
@@ -476,6 +478,15 @@ export function bindHotkeys(
   // Don't forget to remove hotkeys at `removeHotkeys`
 }
 
+function clearSelection(page: Page) {
+  if (!page.root) return;
+  const defaultPageBlock = getDefaultPageBlock(page.root);
+
+  if ('selection' in defaultPageBlock) {
```

Comment:
use `instanceof`

Reply:
The `getDefaultPageBlock` has an incorrect type, I should we can fix it after